README.md

```diff
@@ -65,8 +65,21 @@ Point your browser to: [http://localhost:8000/pages/blockly.html](http://localho
 - Design you own block and then add the metadata to: `blocks` and `generators` in the block repository. An xml snippet should be included in any Javascript file of the block repository, please see [sr_blockly](https://github.com/shadow-robot/sr_blockly/tree/master/sr_blockly_blocks/toolbox) for an example of how to do this.
 - catkin_make will automatically merge all Javascript files into a single file. So in case if you have changed Javascript file you need to run catkin_make.
 
+### Advanced
+
+#### Saving functions
+Functions can be created with the standard in-built function blocks. If you wish to save a function as a new block, select 'Save Function' from the Builder menu. 
+You may select in which category you wish to save the new block, this can be a new category that is specified by appending '/' and the name of the new category to the existing category name, for example:
+
+Select existing category: Variables\
```

Comment:
what does '\' for? and why Or is from capital letter. This sentence would be in the same line.